src/components/SidePanes/UsersPane.scss

```diff
@@ -0,0 +1,3 @@
+.error-msg {
+  color: #FF0000;
```

Comment:
nit: I think typically we've just used bootstrap's color class names, but this whole file might go away if you swap to StatusAlert

Reply:
ty on headsup with status alert component